packages/tracking_repository/lib/src/tracking_repository.dart

```diff
@@ -21,14 +19,10 @@ class TrackingRepository {
 
   /// Updates tracking services accordingly
   void track(TrackEvent event) {
```

Comment:
Swap the event for named parameters instead.

Reply:
I don't think this is a good ideea. This would end up to have all sorts fo strings in the UI side that might need to be reusable.

As before, I advise against this because it weakens encapsulation, scalability and readability.

Please note that if you have a strong opinion on this I can update.

Reply:
I'm aiming at simplifying the API for tracking the event, but hopefully the example I requested on my previous comment will help us decide best.